lib/models/api-key.ts

```diff
@@ -110,10 +110,47 @@ const getApiKeysModel = function (
 				resource: 'api_key',
 				options: mergePineOptions(
 					{
-						// the only way to reason whether
-						// it's a named user api key is whether
-						// it has a name
+						$orderby: 'name asc',
+					},
+					options,
+				),
+			});
+		},
+
+		/**
+		 * @summary Get all named user API keys of the current user
+		 * @name getAllNamedUserApiKeys
+		 * @public
+		 * @function
+		 * @memberof balena.models.apiKey
+		 *
+		 * @param {Object} [options={}] - extra pine options to use
+		 * @fulfil {Object[]} - apiKeys
+		 * @returns {Promise}
+		 *
+		 * @example
+		 * balena.models.apiKey.getAllNamedUserApiKeys().then(function(apiKeys) {
+		 * 	console.log(apiKeys);
+		 * });
+		 *
+		 * @example
+		 * balena.models.apiKey.getAllNamedUserApiKeys(function(error, apiKeys) {
+		 * 	if (error) throw error;
+		 * 	console.log(apiKeys);
+		 * });
+		 */
+		async getAllNamedUserApiKeys(
+			options: BalenaSdk.PineOptions<BalenaSdk.ApiKey> = {},
+		): Promise<BalenaSdk.ApiKey[]> {
+			return await pine.get({
+				resource: 'api_key',
+				options: mergePineOptions(
+					{
 						$filter: {
+							is_of__actor: await sdkInstance.auth.getUserActorId(),
```

Comment:
Do we want to do a check for valid API keys also? (Assuming we have a time check or something similar)

Reply:
We atm don't have such time constrained validity periods, but even if we had I wouldn't embed such a check here.
I mean, this major change was required just b/c we had an embedded check, so the less the better.